lib/parsers/exif_parser.rb

```diff
@@ -41,6 +41,13 @@ def seek(n, seek_mode = IO::SEEK_SET)
   end
 
   class EXIFResult < SimpleDelegator
+    attr_reader :sub_ifds_data
+
+    def initialize(exif_raw_data, sub_ifds_data = {})
```

Comment:
could we by any chance standardize the naming? you pass them in as `raw_exif_data, sub_ifds_hash` but they become `exif_raw_data, sub_ifds_data`. Would be nice to have them everywhere referenced in the same way 😊